### PR TITLE
create mercedes e350 (2010) dbc

### PR DESCRIPTION
**Pull request by quillford**
this is the start of a dbc for the 2010 mercedes benz e350

### CODE REVIEW

File: mercedes_benz_e350_2010.dbc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+BU_: XXX
+
+
+BO_ 3 NEW_MSG_1: 8 XXX
+ SG_ STEERING_ANGLE : 7|32@0- (1,0) [0|4294967295] "" XXX
+
+BO_ 5 NEW_MSG_2: 8 XXX
+ SG_ BRAKE_POSITION : 23|16@0+ (1,0) [0|65535] "" XXX
+ SG_ BRAKE_PRESSED? : 0|8@1+ (1,0) [0|17] "" XXX
```
Comment (rbiasini) [inline mercedes_benz_e350_2010.dbc]: would remove the question mark

Comment (quillford) [reply mercedes_benz_e350_2010.dbc]: fixed

File: mercedes_benz_e350_2010.dbc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+BO_ 3 NEW_MSG_1: 8 XXX
+ SG_ STEERING_ANGLE : 7|32@0- (1,0) [0|4294967295] "" XXX
+
+BO_ 5 NEW_MSG_2: 8 XXX
+ SG_ BRAKE_POSITION : 23|16@0+ (1,0) [0|65535] "" XXX
+ SG_ BRAKE_PRESSED : 0|8@1+ (1,0) [0|17] "" XXX
+
+BO_ 69 NEW_MSG_3: 8 XXX
+ SG_ TURN_SIGNAL_LEVER : 16|8@1+ (1,0) [0|255] "" XXX
```
Comment (rbiasini) [inline mercedes_benz_e350_2010.dbc]: Can we have consistent endianess? some signals are little and some ore big endianess

**Comment by nachoweb** [comment]
Hello, I bought a panda, and soon will be able to test it in my merceces C220 bluetec (2014).
Do you think I should start looking at this dbc file? I am a little lost

**Comment by quillford** [comment]
once  you have a drive in cabana, you can load this dbc and see if anything matches your car. if not you’ll have to make your own.

**Pull request merged**